app/models/payments/ChargeType.scala

```diff
@@ -276,7 +276,9 @@ object ChargeType {
       true
     } catch {
       case ex: IllegalArgumentException =>
+        //$COVERAGE-OFF$
```

Comment:
I wonder if this logger message was flagged up due to the fact we are not using our shared application logger here, but instead creating a new logger inside of the `ChargeType` object? Can you make the object extend `LoggerUtil` and see if the coverage complaint goes away? Even if not, it's good for us to reuse our existing logger anyway.